config/prow/plugins.yaml

```diff
@@ -947,6 +947,15 @@ plugins:
   kubernetes-sigs/security-profiles-operator:
   - release-note
 
+  kubernetes-sigs/k8s-container-image-promoter:
+  - blockade
```

Comment:
blockade, milestone, milestoneapplier, require-matching-label plugins would also need be specifically configured for the repo.

For example, for milestone_applier, you'll need to tell what milestones to apply:

https://github.com/kubernetes/test-infra/blob/12c654eaf2c5aa900963d63233df08e2fcb91d94/config/prow/plugins.yaml#L327-L335

Reply:
ahh, thanks! I think we can drop those, for now, don't think this is a hard requirement for the repo


thanks for your review @nikhita